docs/docs/tutorial/chapter6/comment-form.md

```diff
@@ -253,6 +253,11 @@ const CREATE = gql`
     }
   }
 `
+
+interface FormValues {
+  name: string
+  comment: string
+}
```

Comment:
```suggestion
// highlight-end

// highlight-start
interface FormValues {
  name: string
  comment: string
}
```

I like to break up the highlighting between different parts of the code when they're not closely related. Here the gql template and the FormValues interface are two separate things, so I'd have two separate highlighting blocks.